Solgema/fullcalendar/browser/icalexport.py

```diff
@@ -63,7 +64,7 @@ def update(self):
                 query = {'object_provides': IEvent.__identifier__}
             else:
                 query = {'portal_type': 'Event'}
-            self.events = context.queryCatalog(**query)
+            self.events = api.content.find(context=context, **query)
```

Comment:
Ty for that info!

With context.queryCatalog i get an unknown parameter 'object_provides' or 'portal_type', do you know howto solve that @frisi ?